plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -102,6 +167,15 @@ class ContainerServiceFormSection extends Component {
             <FieldError>{errors.disk}</FieldError>
           </FormGroup>
         </div>
+        {this.getArtifactsInputs(data.fetch)}
+        <FieldError>{errors.fetch}</FieldError>
```

Comment:
Do we need this error if we are already showing them inside the jsx from `this.getArtifactsInput`?